projects/plugins/jetpack/modules/custom-css.php

```diff
@@ -26,6 +26,8 @@ function jetpack_load_custom_css() {
  * Enable CSS module.
  */
 function custom_css_loaded() {
+	_deprecated_hook( 'custom_css_loaded', 'jetpack-$$next-version$$', 'WordPress Custom CSS', 'Jetpack no longer supports Custom CSS. Read the WordPress.org documentation to learn how to apply custom styles to your site: https://wordpress.org/documentation/article/styles-overview/#applying-custom-css' );
```

Comment:
The `_deprecated_hook` is useful here as it lets us define an optional message (unlike `_deprecated_function`, for instance).

Reply:
Should we make this message translatable? It may prove useful to raise the attention of non english speakers?